packages/@lwc/template-compiler/src/codegen/helpers.ts

```diff
@@ -260,8 +262,17 @@ function isStaticNode(node: BaseElement, apiVersion: APIVersion): boolean {
             !(
                 isAllowedFragOnlyUrlsXHTML(nodeName, name, namespace) &&
                 isFragmentOnlyUrl(value.value as string)
-            )
-        );
+            );
+        const isStaticSafeExpression =
+            isExpression(value) &&
+            name !== 'slot' &&
+            // TODO [#3624]: Enable class and style for static content optimization
+            name !== 'class' &&
+            name !== 'style' &&
+            // TODO [#3624]: Revisit whether svgs can be included in static content optimization
+            // svg href needs sanitization.
+            !isSvgUseHref(nodeName, name, namespace);
```

Comment:
I'm not sure if svgs require any special handling in addition to what is provided by LWC so I left them out. From looking at the code, it looks like there may be an additional process to sanitize the html.

Reply:
`tabindex`, `id`, id-referencing attrs, and `isAllowedFragOnlyUrlsXHTML`-related attrs are also not safe, correct? See `bindAttributeExpression` above.

Reply:
To close the loop on this, we reviewed these changes internally. It looks like they should be safe to incorporate into static content optimization.